{
    "type": "bundle",
    "id": "bundle--3726b45d-eb6f-492c-b363-9c9c6999a529",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--04fa0792-6e49-4972-b513-7ed89397814a",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-12-19T00:39:13.071896Z",
            "modified": "2024-12-19T00:39:13.071896Z",
            "name": "CVE-2024-53270",
            "description": "Envoy is a cloud-native high-performance edge/middle/service proxy. In affected versions `sendOverloadError` is going to assume the active request exists when `envoy.load_shed_points.http1_server_abort_dispatch` is configured. If `active_request` is nullptr, only onMessageBeginImpl() is called. However, the `onMessageBeginImpl` will directly return ok status if the stream is already reset leading to the nullptr reference. The downstream reset can actually happen during the H/2 upstream reset. As a result envoy may crash. This issue has been addressed in releases 1.32.3, 1.31.5, 1.30.9, and 1.29.12. Users are advised to upgrade. Users unable to upgrade may disable `http1_server_abort_dispatch` load shed point and/or use a high threshold.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-53270"
                }
            ]
        }
    ]
}